api/Element.json

```diff
@@ -6643,7 +6647,8 @@
             },
             "samsunginternet_android": [
               {
-                "version_added": "3.0"
+                "version_added": "3.0",
+                "notes": "From version 16, returns a promise instead of <code>undefined</code>. The behavior reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Comment:
```suggestion
                "notes": "From version 16, returns a promise instead of <code>undefined</code>. The behavior reflects <a href='https://github.com/w3c/pointerlock/pull/49'>a proposed specification change</a>."
```